dataactvalidator/config/sqlrules/fabs2_detached_award_financial_assistance_1.sql

```diff
@@ -5,18 +5,17 @@ SELECT
     fain,
```

Comment:
For some weird reason it won't let me comment higher, the comment at the top of this file should be updated as well.